common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -83,6 +87,11 @@
   UNEXPECETD_JOURNAL_ENTRY("Unexpected entry in journal: {0}"),
   UNKNOWN_ENTRY_TYPE("Unknown entry type: {0}"),
 
+  // file
+  FILE_ALREADY_EXISTS("{0} already exists"),
+  FILE_CREATE_IS_DIRECTORY("{0} already exists. Directories cannot be overwritten with create"),
+  HDFS_FILE_NOT_FOUND("File {0} with FID {1} is not found."),
```

Comment:
Unless I am mistaken, I don't think "FID" is a commonly used term in tachyon? I think it should just be "id".

Also, should the trailing period be removed? The other messages don't have the period.